tests/test_citations.py

```diff
@@ -0,0 +1,180 @@
+import datetime
+from nose.tools import *
+
+from scripts import parse_citation_styles
+from website import citations
+from website.util import api_url_for
+from website.citations.utils import datetime_to_csl
+from website.models import Node, User
+
+from tests.base import OsfTestCase
+from tests.factories import ProjectFactory, UserFactory
+from tests.test_features import requires_csl_styles
+
+
+bibtex_template = '''@misc{{{id},
+  title={{{title}}},
+  url={{{url}}},
+  publisher={{Open Science Framework}},
+  author={{{authors}}},
+  year={{{year}}},
+  month={{{month}}}
+}}'''
+
+
+class CitationsTestCase(OsfTestCase):
+    def setUp(self):
+        super(CitationsTestCase, self).setUp()
+        self.node = ProjectFactory()
+
+    def tearDown(self):
+        super(CitationsTestCase, self).tearDown()
+        Node.remove()
+
+    @requires_csl_styles
+    def test_render_bibtex(self):
+        """render a node citation as BibTeX"""
```

Comment:
Prefer long method names and comments to docstrings in tests (source: http://cosdev.readthedocs.org/en/latest/process/testing.html#general-testing-guidelines). We do this mostly because `nose` and `pytest` will output docstrings (if they exists) rather than the the test method name when failures occur, which makes it harder to locate failing tests.

Will make this change when I merge.